azure_jump_box.sh

```diff
@@ -8,8 +8,8 @@ sudo chmod +x kubectl
 sudo mv kubectl /usr/local/bin/
 kubectl version
 
-wget https://get.helm.sh/helm-v3.3.4-linux-amd64.tar.gz
-tar -xvf helm-v3.3.4-linux-amd64.tar.gz
+wget https://get.helm.sh/helm-v3.13.1-linux-amd64.tar.gz
```

Comment:
@mehul-lb wondering if we need a similar script to setup a windows jumpbox.

Reply:
Yes if we can install using powershell script. I will explore if we can do that.  With Linux advantage is we are able to configure ssh tunnel access.